src/components/Login/Login.js

```diff
@@ -62,6 +81,8 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
     return values;
   };
 
+  const errorMessage = useMemo(() => getForgotErrorMessage(location), [location]);
```

Comment:
`formMessage`? `initialMessage`?, also `getMessageFromUrl`?

```suggestion
  const formMessage = useMemo(() => getMessageFromUrl(location), [location]);
```

Reply:
fixed